portlet-parent/wicketstuff-portlet/src/main/java/org/apache/wicket/RestartResponseAtInterceptPageExceptionMapper.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.wicket;
+
+import org.apache.wicket.request.IRequestMapper;
+
+/**
+ * @author Konstantinos Karavitis
+ */
+public class RestartResponseAtInterceptPageExceptionMapper
```

Comment:
I will remove this class because I will follow your advice for customizing the SystemMapper.